mailserver/mailserver_test.go

```diff
@@ -559,3 +599,8 @@ func generateEnvelope(sentTime time.Time) (*whisper.Envelope, error) {
 
 	return env, nil
 }
+
+func generateEnvelope(sentTime time.Time) (*whisper.Envelope, error) {
```

Comment:
It looks like it generates a message with a symmetric key, right? Is it used anywhere?

Reply:
This was the default implementation, I've kept it to be compatible with the previous tests. And yes, it's used 4/5 times on the same file.

Reply:
@adambabik I can create some kind of global variable like `h := crypto.Keccak256Hash([]byte("test sample data"))` and directly use `generateEnvelopeWithKeys` everywhere instead if you feel more comfortable with it.

Reply:
Oh if it's used in many places then let's keep it.